apps/modernization-ui/src/pages/patient/search/result/PatientResult.tsx

```diff
@@ -119,7 +119,7 @@ const ResultItem = ({ label, orientation = 'horizontal', children }: ResultItemP
 type ResultItemLabelProps = { children: string };
 
 const ResultItemLabel = ({ children }: ResultItemLabelProps) => (
-    <span className="patient-search-result-item-label">{children}</span>
+    <span className="patient-search-result-item-label card-header-label">{children}</span>
```

Comment:
suggestion:  This is more of a `search-result-item-label` than a `card-header-label`.  PatientResult is defining the `ResultItemLabel` component for this already.  It can be extracted from there and used in all of the search result components.